src/db/core.ts

```diff
@@ -347,7 +347,7 @@ export class SQLFragment<RunResult = pg.QueryResult['rows'], Constraint = never>
 
       } else if (typeof expression.cast === 'string') {
         result.values.push(expression.value);
-        result.text += `CAST(${placeholder} AS "${expression.cast}")`;
+        result.text += `CAST(${placeholder} AS ${expression.cast})`;
```

Comment:
Not sure I follow what removing the quotes fixes. Can you elucidate? Quickly checking in pgAdmin query tool seems like the quotes are indeed not needed, but also not harmful there.

Reply:
With the quotes I was getting errors.. except when the value was "json." All other quoted type strings would fail.

Reply:
Can't argue with empiricism!